src/index.js

```diff
@@ -78,6 +79,7 @@ const MegaFlow = ({
       stepIndex={currentStep}
       stepFormData={stepFormData}
       defaultValues={defaultValues}
+      currenciesFromDB={currenciesFromDB}
```

Comment:
Let's say we want to insert to MegaFlow available languages, VAT rates or even cryptocurrency. Will you also add them as separate props? Sooner or later we would have a component that requires 20 props to render. 

I think adding a new prop is not the best way we can solve it. 
I'd prefer to insert currencies as `defaultValues` or the same way as users are inserted to the fee component.